### ISSUE TITLE
Line number are not reported for classes found in Jade #[] syntax blocks

### ISSUE THREAD

**Issue by benedfit** [jadejs/jade-parser#4]
@ForbesLindesay thanks for fixing #3, but just noticed that I forgot to mention that line numbers and source files aren't reported for interpolated tags either. See:

```json
{
    "type": "Tag",
    "name": "a",
    "selfClosing": false,
    "block": {
        "type": "Block",
        "nodes": []
    },
    "attrs": [{
        "name": "class",
        "val": "\'rho\'",
        "escaped": false
    }, {
        "name": "href",
        "val": "\'#\'",
        "escaped": true
    }, {
        "name": "class",
        "val": "\'rho--modifier\'",
        "escaped": true
    }],
    "attributeBlocks": [],
    "isInline": true,
    "line": 1
}
```

**Comment by benedfit** [comment]
This has been fixed post 0.0.5

**Comment by benedfit** [comment]
Actually scratch that, the line numbers are still incorrectly reported as of 0.0.8
